draft-secure-credential-transfer.md

```diff
@@ -203,9 +203,56 @@ Sender device may terminate the secure credential transfer by deleting the mailb
 ~~~
 {: #stateful-flow-image title="Sample stateful workflow"}
 
+## Provisioning Information Structure
+
+The Provisioning Information is the data transfered via the Relay Server between the Sender device and Receiver device. Each use case defines its own specalized Provisioning Information format, but all formats must at least ahead to the following structure.
+
+| Key    | Type   | Description 
+| ------ | ------ | --- 
+| format | String | The Provisioning Information format that the message follows. This is used by the Sender device and Receiver device to know how to parse the message.
```

Comment:
Are we also planning on defining the genericSharingData to which the format is applied to?

Reply:
Yes, updated to add that field.